README.md

```diff
@@ -15,17 +15,35 @@ suitable for more general usage.
 
 # Status
 
-Bazel builds can be run from the [llvm-bazel subdirectory](./llvm-bazel) for a
-build based on a submodule and [http-archive-demo](./http-archive-demo) for a
-build based on
+MLIR builds with Cuda and Vulkan support disabled.
+
+Building LLVM core has many issues with finding `.inc` files. These issues also
```

Comment:
My own opinion: I think it might be better to leave off comparisons to Tensorflow since it should be irrelevant once this is somewhat complete.

Reply:
Fair point. Done.